design/namedspaces.go

```diff
@@ -24,4 +24,22 @@ var _ = a.Resource("namedspaces", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.NotFound, JSONAPIErrors)
 	})
+
+	a.Action("list", func() {
+		a.Routing(
+			a.GET("/:userName"),
+		)
+		a.Description("List spaces owned by a user.")
+		a.Params(func() {
+			a.Param("userName", d.String, "User name of the owner of the space")
+			a.Param("page[offset]", d.String, "Paging start position")
+			a.Param("page[limit]", d.Integer, "Paging size")
+		})
+
+		a.Response(d.OK, func() {
+			a.Media(spaceList)
+		})
+		a.Response(d.BadRequest, JSONAPIErrors)
+		a.Response(d.InternalServerError, JSONAPIErrors)
```

Comment:
Added. Good catch. Without that the server did not respond with 404s even though the controlled instructed it to.